src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.html

```diff
@@ -3,34 +3,45 @@
     <div class="row">
       <div class="ubs_profile-header">
         <div class="header-text">
-          <h3>{{ 'ubs-client-profile.data' | translate }}</h3>
-          <ng-container *ngIf="!isEditing">
-            <button tabindex="0" class="btn btn-outline-success open" (click)="openChangePasswordDialog()">
-              {{ 'ubs-client-profile.btn.change-password' | translate }}
-            </button>
-            <button tabindex="0" class="btn btn-outline-success delete" (click)="openDeleteProfileDialog()">
-              {{ 'ubs-client-profile.btn.delete-profile' | translate }}
-            </button>
-          </ng-container>
+          <div class="header-title">
+            <h3>{{ 'ubs-client-profile.data' | translate }}</h3>
+          </div>
+          <div class="header-buttons">
+            <ng-container *ngIf="!isEditing">
+              <button tabindex="0" class="btn btn-outline-success open" (click)="openChangePasswordDialog()">
+                {{ 'ubs-client-profile.btn.change-password' | translate }}
+              </button>
+              <button tabindex="0" class="btn btn-outline-success delete" (click)="openDeleteProfileDialog()">
+                {{ 'ubs-client-profile.btn.delete-profile' | translate }}
+              </button>
+            </ng-container>
+          </div>
         </div>
       </div>
+
       <div role="form" tabindex="0" id="form" class="input-group input-group-sm">
         <form *ngIf="!isFetching; else spinner" [formGroup]="userForm">
-          <h5 [ngClass]="{ h5MarginBottom: isEditing }">{{ 'ubs-client-profile.personal' | translate }}</h5>
+          <h5 [ngClass]="{ h5MarginTop: isEditing }">{{ 'ubs-client-profile.personal' | translate }}</h5>
           <div class="personal inline">
-            <div [ngClass]="{ inputMargin: isEditing }" class="form-group col-sm-6">
+            <div class="form-group inputMargin">
```

Comment:
CSS classes' spelling is usually kebab-cased. Feel free to check out this article: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/